python/pipeline/meso.py

```diff
@@ -478,16 +478,29 @@ class SummaryImages(dj.Computed):
 
     -> MotionCorrection
     -> shared.Channel
-    ---
-    average             : longblob          # l6-norm (across time) of each pixel
-    correlation         : longblob          # (average) temporal correlation between each pixel and its eight neighbors
     """
 
     @property
     def key_source(self):
         # Run make_tuples once per scan iff MotionCorrection is done
         return ScanInfo() & MotionCorrection() & {'meso_version': CURRENT_VERSION}
 
+    class Average(dj.Part):
```

Comment:
Can you give the attributes different names than `image`? It becomes very annoying for joining.